README.md

```diff
@@ -40,55 +40,87 @@ In your service worker you just need to use `importScripts` to load Shed
 importScripts('bower_components/shed/shed.js'); // Update path to match your own setup
 ```
 
-## API
-
+## Basic usage
```

Comment:
Can you clarify that all of the following examples need to live within the context of a service worker (not in a controlled page)? Say something like:

"Within `my-service-worker.js`..."